maas/client/viscera/machines.py

```diff
@@ -174,6 +177,8 @@ class Machine(Object, metaclass=MachineType):
         :param hwe_kernel: The HWE kernel to deploy. Probably only relevant
             when deploying Ubuntu.
         :param comment: A comment for the event log.
+        :param wait: If specified, wait until the deploy is complete, polling
+            at the specified frequency.
```

Comment:
I don't think `wait=True` is redudant. The default wait_interval should always be 5. But your saying that you want to wait.